elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -203,24 +204,28 @@ default void preCommit() {
             EntityDictionary dictionary,
             Set<Predicate> filters,
             Sorting sorting,
-            Pagination pagination
+            Pagination pagination,
+            String relationNameForSortAndPage
```

Comment:
These are breaking changes.  We don't want to break the contract until Elide 3.0